app/views/batch_contexts/show.html.erb

```diff
@@ -1,3 +1,49 @@
 <div class="container mt-5">
-  <%= render @batch_context %>
+  <div class="row">
```

Comment:
we do not need this "batch context detail" partial anymore -- it is now just used once